src/sentry/lang/java/processing.py

```diff
@@ -23,3 +33,185 @@ def deobfuscate_exception_value(data):
         )
 
     return data
+
+
+def _merge_frame_context(new_frame, symbolicated):
```

Comment:
Can you add types to each function, and enable strict type checking using pyproject.toml?